### ISSUE TITLE
how to use

### ISSUE THREAD

**Issue by kdurant** [linuxbest/ahci_mpi#1]
hello 
     I want to  impletement ATA to SATA interface?

```
 Can I use your project?
```

**Comment by linuxbest** [comment]
Yes, Which device you want support? Now I has been tested it Virtex5 FXT,
spartan6 GTP, kintex7 GTX.

On Mon, Sep 16, 2013 at 8:22 PM, kdurant notifications@github.com wrote:

> hello
> I want to impletement ATA to SATA interface?
> 
>  Can I use your project?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/linuxbest/ahci_mpi/issues/1
> .

## 

Hu Gang

**Comment by kdurant** [comment]
I want to use Altera
I'm not familiar xilinx

**Comment by linuxbest** [comment]
Many work need to do for support altera device, which altera device you
want use?

You want connect a SATA device to old ATA only board, right?

thanks.

On Tue, Sep 17, 2013 at 12:17 AM, kdurant notifications@github.com wrote:

> I want to use Altera
> I'm not familiar xilinx
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/linuxbest/ahci_mpi/issues/1#issuecomment-24568310
> .

## 

Hu Gang
